templates/CRM/Event/Form/Registration/AdditionalParticipant.tpl

```diff
@@ -41,7 +41,9 @@
     </div>
 {/if}
 
-{include file="CRM/UF/Form/Block.tpl" fields=$additionalCustomPre}
+<div class="crm-public-form-item crm-section custom_pre-section" id="CustomPreBlock">
```

Comment:
Can we leave off the id? I think ids are something we could use less of in our markup rather than more.